### DISCUSSION TITLE
How to retreive color data on init?

### DISCUSSION THREAD

**Discussion by Potato22** [mdbassit/Coloris#161] [Q&A] [answered]
Coloris seem to remember the last color selected by the user on load, but how can you retrieve the data on load? There is no event for "onLoad" as far as I know, I'd appreciate your help on this, thank you.

**Comment by mdbassit** [comment, marked as answer]
Coloris doesn't have a state at all so it can't remember anything. All it does is render the color value that was already set in the input field it's attached to.

**Comment by Potato22** [comment]
I see, I may have realized that I'm using a dev server that hotswaps the DOM instead of refreshing entirely which made it look like it remembers the last state. Apologies.
